charon-ml/src/GAstOfJson.ml

```diff
@@ -912,6 +912,9 @@ and literal_of_json (js : json) : (literal, string) result =
     | `Assoc [ ("Scalar", scalar) ] ->
```

Comment:
As indicated at the top of this file, it should not be modified by hand. Edit `charon/src/bin/generate-ml/templates/GAstOfJson.ml` instead and call `make generate-ml` to propagate your changes to this file. In this particular case the definition is auto-generated so you shouldn't have anything to do.